rose/common/obstacles.py

```diff
@@ -2,12 +2,12 @@
 
 import random
 
-NONE    = ""         # NOQA
-CRACK   = "crack"    # NOQA
-TRASH   = "trash"    # NOQA
+NONE = ""  # NOQA
+CRACK = "crack"  # NOQA
+TRASH = "trash"  # NOQA
 PENGUIN = "penguin"  # NOQA
-BIKE    = "bike"     # NOQA
-WATER   = "water"    # NOQA
+BIKE = "bike"  # NOQA
+WATER = "water"  # NOQA
 BARRIER = "barrier"  # NOQA
```

Comment:
The prev one looks better IMHO

Reply:
Yes but black does not agree, again we have to accept black's style.